static/app/views/issueDetails/groupActivityItem.tsx

```diff
@@ -23,58 +23,71 @@ import {
 type Props = {
   activity: GroupActivity;
   author: React.ReactNode;
-  orgSlug: Organization['slug'];
+  organization: Organization;
   projectId: Project['id'];
 };
 
-function GroupActivityItem({activity, orgSlug, projectId, author}: Props) {
-  const issuesLink = `/organizations/${orgSlug}/issues/`;
+function GroupActivityItem({activity, organization, projectId, author}: Props) {
+  const issuesLink = `/organizations/${organization.slug}/issues/`;
+  const hasEscalatingIssuesUi = organization.features.includes('escalating-issues-ui');
 
   function getIgnoredMessage(data: GroupActivitySetIgnored['data']) {
+    const ignoredOrArchived = hasEscalatingIssuesUi ? t('archived') : t('ignored');
     if (data.ignoreDuration) {
-      return tct('[author] ignored this issue for [duration]', {
+      return tct('[author] [action] this issue for [duration]', {
         author,
+        action: ignoredOrArchived,
         duration: <Duration seconds={data.ignoreDuration * 60} />,
       });
     }
 
     if (data.ignoreCount && data.ignoreWindow) {
       return tct(
-        '[author] ignored this issue until it happens [count] time(s) in [duration]',
+        '[author] [action] this issue until it happens [count] time(s) in [duration]',
         {
           author,
+          action: ignoredOrArchived,
           count: data.ignoreCount,
           duration: <Duration seconds={data.ignoreWindow * 60} />,
         }
       );
     }
 
     if (data.ignoreCount) {
-      return tct('[author] ignored this issue until it happens [count] time(s)', {
+      return tct('[author] [action] this issue until it happens [count] time(s)', {
         author,
+        action: ignoredOrArchived,
         count: data.ignoreCount,
       });
     }
 
     if (data.ignoreUserCount && data.ignoreUserWindow) {
       return tct(
-        '[author] ignored this issue until it affects [count] user(s) in [duration]',
+        '[author] [action] this issue until it affects [count] user(s) in [duration]',
         {
           author,
+          action: ignoredOrArchived,
           count: data.ignoreUserCount,
           duration: <Duration seconds={data.ignoreUserWindow * 60} />,
         }
       );
     }
 
     if (data.ignoreUserCount) {
-      return tct('[author] ignored this issue until it affects [count] user(s)', {
+      return tct('[author] [action] this issue until it affects [count] user(s)', {
```

Comment:
Perhaps out of the scope of this PR, but it's not clear from this wording whether this means "ignore until it's affected _x_ users in total" or "ignore until it affects another _x_ users."